src/main/java/ch/wisv/areafiftylan/users/controller/UserRestController.java

```diff
@@ -148,6 +149,15 @@ public User getUserById(@PathVariable Long userId) {
         return createResponseEntity(HttpStatus.OK, "User disabled");
     }
 
+    @PreAuthorize("hasRole('ADMIN')")
+    @GetMapping("/{userID}/agecheck")
```

Comment:
I assume this is a typo and should be `/{userId}/agecheck`, should it not? It does not match the method argument right now.